samples/simulated_device_cloud_upload/README.md

```diff
@@ -65,40 +65,40 @@ We recommend using the Azure recipes provided by Intel available in the
 [meta-iot-cloud](https://github.com/intel-iot-devkit/meta-iot-cloud) repository.
 Follow the instructions in the 
 [README](https://github.com/intel-iot-devkit/meta-iot-cloud#configuration) to 
-build an Edison image with the azure-iot-gateway-sdk libraries included.
+build an Edison image with the iot-edge libraries included.
 
 #### Intel Edison
 
 The Edison does not support all of the Azure recipes available in the "meta-iot-cloud" repository. To get the Edison image to build, we have to limit the recipes included in the image.
 
-- Remove the azure-iot-gateway-sdk-java target.
```

Comment:
We need to coordinate with Intel to change stuff in their meta-iot-cloud repo before we can change these instructions.